test/libsolidity/smtCheckerTests/operators/delete_function_type_1.sol

```diff
@@ -1,16 +1,16 @@
 struct S {
-    function() e;
+    function() external e;
 }
 
 contract C {
     S s;
-    function() f;
+    function() external f;
 
     constructor() {
         delete s.e;
         assert(s.e == f); // should fail for now because function pointer comparisons are not supported
     }
 }
 // ----
-// Warning 7229: (128-136): Assertion checker does not yet implement the type function () for comparisons
-// Warning 6328: (121-137): CHC: Assertion violation happens here.
+// Warning 7229: (146-154): Assertion checker does not yet implement the type function () external for comparisons
+// Warning 6328: (139-155): CHC: Assertion violation happens here.\nCounterexample:\ns = {e: 0}, f = 0\n\nTransaction trace:\nC.constructor()
```

Comment:
By the way - these appeared the last time around as well when I updated the tests, and initially I thought something was broken, so I removed them manually, re-ran, and the tests were passing just fine. However, I now see that most of the SMT tests in fact have these counter examples, so I'll leave them in.

Reply:
Not sure, could have something do do with #14446. Or maybe it could not produce a counterexample for internal comparison but for external it can.

As long as the counterexample is not wrong, I'd not worry about it.